app/code/Magento/InventorySales/Setup/InstallSchema.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventorySales\Setup;
+
+use Magento\Framework\Setup\InstallSchemaInterface;
+use Magento\Framework\Setup\ModuleContextInterface;
+use Magento\Framework\Setup\SchemaSetupInterface;
+use Magento\InventorySales\Setup\Operation\CreateStockChannelTable;
+
+class InstallSchema implements InstallSchemaInterface
+{
+    /**
+     * @var CreateStockChannelTable
+     */
+    private $createStockChannelTable;
+
+    /**
+     * @param CreateStockChannelTable $createStockChannelTable
+     */
+    public function __construct(
+        CreateStockChannelTable $createStockChannelTable
+    ) {
+        $this->createStockChannelTable = $createStockChannelTable;
+    }
+
+    /**
+     * {@inheritdoc}
```

Comment:
use pls `@inheritdoc` without `{}` - {@inheritdoc} is only for DocBlocks  if you must write a more specific description